discretize/DiffOperators.py

```diff
@@ -774,6 +774,37 @@ def aveCC2F(self):
                 ), format="csr")
         return self._aveCC2F
 
+    # TODO: modularize
+    @property
+    def aveCC2FV(self):
+        """
+        Construct the averaging operator on cell cell centers to
+        faces as a vector.
+        """
+        if getattr(self, '_aveCC2FV', None) is None:
+            if self.dim == 1:
+                self._aveCC2FV = self._aveCC2Fx
+            elif self.dim == 2:
+                self._aveCC2Fx = sp.kron(speye(self.nCy), av_extrap(self.nCx))
+                self._aveCC2Fy = sp.kron(speye(self.nCy), av_extrap(self.nCx))
+                self._aveCC2FV = sp.block_diag((
+                        self.aveCC2Fx, self.aveCC2Fy
+                        ), format="csr")
+            elif self.dim == 3:
+                self._aveCC2Fx = kron3(
```

Comment:
sorry to be picky, would you mind closing the bracket on the same level as it the `self`? eg
```
self._aveCC2Fz = kron3(
    some_things
)
```